dev/MRTCore/mrt/mrm/include/mrm/common/file/MrmFiles.h

```diff
@@ -184,12 +184,12 @@ extern "C"
      */
     typedef struct _MRMFILE_ENVIRONMENT_MAPPING_HEADER
     {
-        DEF_CHECKSUM baseEnvironmentChecksum;
-        DEF_CHECKSUM currentEnvironmentChecksum;
-        UINT16 currentMinorVersion;
-        UINT16 numNewQualifiers;
-        UINT16 namesPoolSizeInChars;
-        UINT16 pad;
+        DEF_CHECKSUM baseEnvironmentChecksum = 0;
+        DEF_CHECKSUM currentEnvironmentChecksum = 0;
+        UINT16 currentMinorVersion = 0;
+        UINT16 numNewQualifiers = 0;
+        UINT16 namesPoolSizeInChars = 0;
+        UINT16 pad = 0;
     } MRMFILE_ENVIRONMENT_MAPPING_HEADER;
```

Comment:
is this even allowed?

Reply:
This was to address
![image](https://github.com/microsoft/WindowsAppSDK/assets/48363984/3eab85ea-7cfc-4699-9f3b-547eee3ac551)

But I think I can just use bracket initialization to fix this instead of doing it in the struct

Reply:
But yes I think you can define default values on a struct?